xformers/components/attention/csrc/cuda/mem_eff_attention/kernel_forward.h

```diff
@@ -477,6 +502,7 @@ struct AttentionKernel {
           shared_storage.after_mm0.si, accum, my_lane_id, output_tile_coords);
 
       __syncthreads();
+      auto siacc = shared_storage.after_mm0.si.accum_ref();
```

Comment:
nit: is this debug code, or does `accum_ref()` modify `si` in-place?

Reply:
leftover debug code! Thanks for catching that